language/models/cache.ts

```diff
@@ -200,4 +200,33 @@ export default class Cache {
       })
     }
   }
+
+  static refereneceByOffset(scope: Cache, offset: number): Declaration|undefined {
```

Comment:
```suggestion
  static referenceByOffset(scope: Cache, offset: number): Declaration|undefined {
```

Simple spelling mistake...